### PR TITLE
[nnfwapi] Support RUY backend testing in GenModelTest

### PR DESCRIPTION
**Pull request by periannath**
- This commit supports RUY backend testing in GenModelTest

ONE-DCO-1.0-Signed-off-by: JiHwan Yeo <jihwan.yeo@samsung.com>

### CODE REVIEW

File: tests/nnfw_api/src/GenModelTest.h [hunk 1]
```diff
@@ -228,6 +228,10 @@ class GenModelTestContext
       {
         _backends.push_back(backend);
       }
+      else if (backend == "ruy")
```
Comment (YongseopKim) [inline tests/nnfw_api/src/GenModelTest.h:231]: `if (backend == "cpu" || if (backend == "ruy"))` looks better. I do like this on #5021

Comment (periannath) [reply tests/nnfw_api/src/GenModelTest.h]: @YongseopKim Thanks! It looks better. I've updated.

**Review by wateret — APPROVED** [review]
LGTM

**Review by YongseopKim — APPROVED** [review]
LGTM

**Pull request merged**